src/components/service-metrics/controllers.tsx

```diff
@@ -398,6 +422,33 @@ export async function downloadServiceMetrics(
   let contents: ReadonlyArray<ReadonlyArray<string>>;
 
   switch (serviceLabel) {
+    case 'aws-sqs-queue':
+      const servicePlan = await cf.servicePlan(service.entity.service_plan_guid);
+      const sqsMetricSeries = await new SQSMetricDataGetter(
+        new cw.CloudWatchClient({
+          endpoint: ctx.app.awsCloudwatchEndpoint,
+          region: ctx.app.awsRegion,
+        }),
+      ).getData(
+        [params.metric],
+        params.serviceGUID,
+        period,
+        rangeStart,
+        rangeStop,
+        servicePlan.entity.name,
+      );
+
+      headers = ['Service', 'Time', 'Value'];
+      contents = values(sqsMetricSeries[params.metric])
+        .map(metric =>
+          metric.metrics.map(series => [
+            serviceLabel,
```

Comment:
I think this is wrong, but it's not your fault (you copy pasted)...

It will essentially create a column full of `aws-sqs-queue` which seems pointless.

I am not certain if it should be a `service.name` which also would be a massive duplication, or if this column should/could be deleted xD

I'll bring it up with our PMs.

Reply:
yeah I thought it was bit weird ... kinda makes sense if you download all the CSVs and merge the data together or something :shrug: